media/js/base/mozilla-utils.js

```diff
@@ -45,10 +52,10 @@ if (typeof Mozilla === 'undefined') {
 
     // language switcher
     Utils.initLangSwitcher = function() {
-        var $language = $('#page-language-select');
-        var previousLanguage = $language.val();
-        $language.on('change', function() {
-            var newLanguage = $language.val();
+        var language = document.getElementById('page-language-select');
```

Comment:
While jQuery fails gracefully, this now throws an error on Protocol pages where `language` is undefined, so we may need to add a check that it exists before trying to query it.

```
jQuery.Deferred exception: language is null Utils.initLangSwitcher@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:155:13
@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:1690:9
j@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:29997
g/</k<@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:30313
setTimeout handler*g/<@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:30520
i@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:28012
fireWith@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:28783
fire@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:28817
i@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:28012
fireWith@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:28783
ready@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:31932
S@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:3:33
EventListener.handleEvent*@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:3:147
@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:220
@http://127.0.0.1:8000/media/js/BUNDLES/common-protocol.js:2:2
 undefined jquery-3.2.1.min.js:2
TypeError: language is null[Learn More] mozilla-utils.js:56

    initLangSwitcher mozilla-utils.js:56 <anonymous> base-page-init.js:23 j2jQuery
```